service/src/main/java/bio/terra/catalog/rawls/RawlsService.java

```diff
@@ -0,0 +1,34 @@
+package bio.terra.catalog.rawls;
+
+import bio.terra.catalog.config.RawlsConfiguration;
+import bio.terra.catalog.model.SystemStatusSystems;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public record RawlsService(RawlsConfiguration rawlsConfig) {
+  private static final Logger logger = LoggerFactory.getLogger(RawlsService.class);
+
+  @Autowired
+  public RawlsService {}
```

Comment:
Is this necessary? It's good to see a `record` `Component`, I'd been meaning to try it out.

Reply:
Actually the `record Component` prevents `mockito` from using `spy`, so I had to switch it back to a regular component.

Reply:
That makes sense. Looking at the code again, if we moved the code that created the APIs into the configuration class (and renamed it accordingly), and then mocked that instead, that might be a better approach overall. But that can wait for another PR.